### ISSUE TITLE
For people that where having problems with sndcpy closing itself

### ISSUE THREAD

**Issue by Tonyshii** [rom1v/sndcpy#153]
So, I was having some problems with sndcpy after updating my phone to android 11, at first I thought that the problem was VLC since I was having problems with it for a while now, I tried tweaking the script file to no avail.
But it turns out that the problem was android all along, android in all its "wisdom" was "optimizing" the battery usage of the sndcpy app, even while allowing the app to run in the background it was still killing it for some reason. 
So by disabling the app "optimization", now android doesn't close the app on it's own, and I get continuous sound on my PC from my phone. 

**TL;DR:** android 11 is dumb and closes the sndcpy app, to fix it go to the app info > battery (under usage) > optimize battery use > select "all" from the drop down menu > find sndcpy and disable it. Thats it! now the script works for me (I'm on linux so I have no way to confirm for windows, but I assume it should work too).

Would be nice is other people can try this and confirm it works for them as well. 

(P.S.: I'm new to GitHub so no idea if this can be linked to other issue threads regarding this, would appreciate help with that)

**Comment by WolfeTMGaming** [comment]
> So, I was having some problems with sndcpy after updating my phone to android 11, at first I thought that the problem was VLC since I was having problems with it for a while now, I tried tweaking the script file to no avail.
> But it turns out that the problem was android all along, android in all its "wisdom" was "optimizing" the battery usage of the sndcpy app, even while allowing the app to run in the background it was still killing it for some reason.
> So by disabling the app "optimization", now android doesn't close the app on it's own, and I get continuous sound on my PC from my phone.
> 
> **TL;DR:** android 11 is dumb and closes the sndcpy app, to fix it go to the sndcpy app info > battery (under usage) > optimize battery use > select "all" from the drop down menu > find sndcpy and disable it. Thats it! now the script works for me (I'm on linux so I have no way to confirm for windows, but I assume it should work too).
> 
> Would be nice if other people can try this and confirm it works for them as well.
> 
> (P.S.: I'm new to GitHub so no idea if this can be linked to other issue threads regarding this, would appreciate help with that)

I just disabled this on my phone this very moment on my Samsung Note S20. I am going to test this later on a stream and will return with a confirmation of whether or not this fixes the problem. I am on latest Android and Windows as of right now! 

Thanks for providing a solution and thank you to @rom1v for linking this topic with my topic! I see you're looking out and that's also much appreciated Rom!

**Comment by rickvian** [comment]
Good news, i manage to solve the issue by keeping the app open, 
open task manager in android
and click on the app icon, press "keep open"
![Image 1619](https://user-images.githubusercontent.com/8034141/139517149-bdff27bd-0ab5-40a8-96e8-d3aaaa7a3ab6.jpg)

then it will be locked 

![image](https://user-images.githubusercontent.com/8034141/139517152-6c609f19-8b1d-4bf2-90ae-b0c599d89680.png)

**Comment by cleycley** [comment]
> Good news, i manage to solve the issue by keeping the app open, open task manager in android and click on the app icon, press "keep open" ![Image 1619](https://user-images.githubusercontent.com/8034141/139517149-bdff27bd-0ab5-40a8-96e8-d3aaaa7a3ab6.jpg)
> 
> then it will be locked
> 
> ![image](https://user-images.githubusercontent.com/8034141/139517152-6c609f19-8b1d-4bf2-90ae-b0c599d89680.png)

not work
